docs/css/react.scss

```diff
@@ -122,6 +122,7 @@ h1, h2, h3, h4, h5, h6 {
 .hash-link {
   color: $mediumTextColor;
   display: none;
+  padding-left: 8px;
```

Comment:
Is this a UI change or is this just to compensate for something that's happening in that Redcarpet library? If it is a UI change, would you mind posting a before and after screen shot to show what you're changing here?